get_s3.go

```diff
@@ -127,17 +127,17 @@ func (g *S3Getter) Get(dst string, u *url.URL) error {
 			req.Marker = aws.String(lastMarker)
 		}
 
-		resp, err := client.ListObjectsWithContext(ctx, req)
+		resp, err := client.ListObjects(ctx, req)
```

Comment:
The [S3 API Reference documentation](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjects.html) recommends using `ListObjectsV2` instead of `ListObjects`.

`ListObjectsV2` can also be paginated using `NewListObjectsV2Paginator` instead of manually managing the pagination.

Reply:
@gdavison Thanks. Applied all comments

Reply:
I updated the second reference to ListObjectsV2 changing Marker to Delimiter (I honestly don't know what they mean :-) )